### DISCUSSION TITLE
v1.9.0

### DISCUSSION THREAD

**Discussion by cdubz** [babybuddy/babybuddy#315] [General]
[Full Changelog](https://github.com/babybuddy/babybuddy/compare/v1.8.3...v1.9.0)

**Implemented enhancements:**

- Favicon is hard to see on dark backgrounds [\#297](https://github.com/babybuddy/babybuddy/issues/297)
- Sleep graph y-axis should go "down" from midnight at the top. [\#289](https://github.com/babybuddy/babybuddy/issues/289)
- Allow manual editting of datetime values [\#284](https://github.com/babybuddy/babybuddy/issues/284)
- Add prominent link to timeline view from dashboard [\#271](https://github.com/babybuddy/babybuddy/issues/271)
- Mobile UI Tweaks [\#229](https://github.com/babybuddy/babybuddy/issues/229)
- Adding Portuguese Translation [\#308](https://github.com/babybuddy/babybuddy/pull/308) ([signorecello](https://github.com/signorecello))
- Date picker changes [\#304](https://github.com/babybuddy/babybuddy/pull/304) ([ntrecina](https://github.com/ntrecina))
- reversing y-axis of sleep plot \(top to down\) [\#295](https://github.com/babybuddy/babybuddy/pull/295) ([fodi666](https://github.com/fodi666))

**Fixed bugs:**

- Buttons icon glyphs missing in time selector [\#302](https://github.com/babybuddy/babybuddy/issues/302)
- If there is only one sleep entry and it starts one day and ends the next day the sleep graph excepts. [\#290](https://github.com/babybuddy/babybuddy/issues/290)
- Sleep graph uses 12h format when 24h is enabled. [\#287](https://github.com/babybuddy/babybuddy/issues/287)

**Closed issues:**

- All users lost [\#305](https://github.com/babybuddy/babybuddy/issues/305)
- iOS Shortcuts [\#299](https://github.com/babybuddy/babybuddy/issues/299)
- Button group flex spacing is weird on smaller screens [\#298](https://github.com/babybuddy/babybuddy/issues/298)
- Add a gulp commend for font updates [\#296](https://github.com/babybuddy/babybuddy/issues/296)
- Last Feed Time vs Last Time First Fed [\#162](https://github.com/babybuddy/babybuddy/issues/162)

**Merged pull requests:**

- Add CodeQL workflow config [\#314](https://github.com/babybuddy/babybuddy/pull/314) ([cdubz](https://github.com/cdubz))
- Improve logo/favicon styling [\#312](https://github.com/babybuddy/babybuddy/pull/312) ([cdubz](https://github.com/cdubz))
- Feeding form type breast autoselect [\#307](https://github.com/babybuddy/babybuddy/pull/307) ([ntrecina](https://github.com/ntrecina))
- Gitpod config [\#303](https://github.com/babybuddy/babybuddy/pull/303) ([lutzky](https://github.com/lutzky))
- Nav items linking to timeline from dashboard [\#278](https://github.com/babybuddy/babybuddy/pull/278) ([lutzky](https://github.com/lutzky))

<hr /><em>This discussion was created from the release <a href='https://github.com/babybuddy/babybuddy/releases/tag/v1.9.0'>v1.9.0</a>.</em>

**Comment by LostSoulfly** [comment]
I can't believe you're still working on this! I'll be setting it up again in a few months for the second time ;)

**Reply by cdubz** [reply]
Congrats 🎉

This got us through two kids and we’re not having any more but other people keep using the app so I’ll keep hacking away at it 🤷

Also lots of quality contributions from others lately!
